.gitlab-ci.yml

```diff
@@ -0,0 +1,56 @@
+stages:
+    - build
+    - deploy
+
+.maven-build:
+    image: maven:3.5.4-jdk-8
+    stage: build
```

Comment:
stage this job (in this case, the children job of this) will be executed in. All the jobs in the same stage are executed in parallel, unless you define dependencies between them, to make them run sequentially.